elementary/monitor/alerts/alert.py

```diff
@@ -31,13 +32,15 @@ def __init__(
         self.elementary_database_and_schema = elementary_database_and_schema
         self.detected_at_utc = None
         self.detected_at = None
+        self.detected_at_utc_str = None
```

Comment:
What do you think about having a single `detected_at` which is a `datetime` object with timezone included and format it whenever needed? Seems a bit exaggerated to have 4 different class properties which are roughly the same thing.

Reply:
You are right.